core/src/main/java/org/elasticsearch/action/admin/indices/shards/IndicesShardStoresResponse.java

```diff
@@ -115,9 +116,10 @@ private void writeTo(StreamOutput out) throws IOException {
         private StoreStatus() {
         }
 
-        public StoreStatus(DiscoveryNode node, long version, Allocation allocation, Throwable storeException) {
+        public StoreStatus(DiscoveryNode node, long version, String allocationId, Allocation allocation, Throwable storeException) {
```

Comment:
I liked allocatedAs better :) (note that the name of the java JSON field is ALLOCATED :))